### PR TITLE
update build scripts

### PR DESCRIPTION
**Pull request by yifever**
main changes to install_piper:
- use source instead of directly calling it to update path correctly. this means no more inputs with read, so just use args instead

main changes to build:
- fix server build
- fix skipping download also skipping config
- move AUDIO_LISTENER_SCRIPT setup to during config and set record to default (sample seems to be broken)

main changes to README:
- add command for piper
- add hint on where the server is build

### CODE REVIEW

File: install_piper.sh [hunk 1]
```diff
@@ -1,78 +1,94 @@
...
-set -e
-
-# Determine system architecture
-ARCH=$(uname -m)
+function main() {
+     
+    # Check for help flag
+    if [[ "$@" == "--help" || "$@" == "-h" ]]; then
+      echo "Usage: source install_piper.sh [download] [model]"
+      echo "   download: Set to true to download. Default is true."
```
Comment (ghost) [inline install_piper.sh]: we should keep default as false, don't want users disk to populate unncessarily. Hence the prompt later to ask permission explicitly. Kinda like what other installers do

File: build.sh [hunk 2]
```diff
@@ -73,10 +73,23 @@ cd ../
 cd llama.cpp
 
 echo "Compiling llama.cpp server"
-LLAMA_BUILD_SERVER=1 make
+if [[ $(uname -m) == "arm64" ]]; then
```
Comment (ghost) [inline build.sh:76]: Thank you!

File: install_piper.sh (hunk 1 above)
Comment (yifever) [reply install_piper.sh]: changed

**Review by ghost — APPROVED** [review]
Thanks for doing this!

**Pull request merged**